samples/cpp/benchmark_app/utils.hpp

```diff
@@ -5,159 +5,110 @@
 #pragma once
 
 #include <map>
+#include <openvino/openvino.hpp>
+#include <samples/slog.hpp>
 #include <string>
 #include <vector>
 
 namespace benchmark_app {
 struct InputInfo {
-    InferenceEngine::Precision precision;
-    InferenceEngine::SizeVector shape;
-    std::string layout;
+    ov::element::Type precision;
+    ov::PartialShape partialShape;
+    ov::Shape dataShape;
+    ov::Layout layout;
+    ov::Layout originalLayout;
```

Comment:
As far as I understood `originalLayout` is the layout of input from the network. Actually command line parameter `layout` means the same thing. We don't need to keep both. We need to apply `layout` parameter value using API 2.0 and use a single value.

We also can think about the introduction `tensor_layout` to apply to preprocess, but it is a separate feature.

Reply:
original layout is removed. It was workaround for previous version because some model were incorrectly converted (without setting proper layout). In API 2.0 it should not be a problem any more as we specify layout explicitly.